Dockerfile

```diff
@@ -10,12 +10,19 @@ RUN dnf update -y && \
         libguestfs-devel && \
     dnf clean -y all
 
-RUN mkdir -p /output && \
-    cd /output && \
-    libguestfs-make-fixed-appliance --xz && \
-    KERNEL_VERSION=$(rpm -qa kernel-core | sed 's/kernel-core-\(.*\)\.el9.*/\1/') && \
+# Create tarball for the appliance. This fixed libguestfs appliance uses the root in qcow2 format as container runtime not always handle correctly sparse files. This appliance can be extracted and copied directly in the container image.
+RUN mkdir /output && \
```

Comment:
Either retain the `-p` when creating `/output` (cleaner diff) or avoid it when creating `/appliance`.

There should be no functional difference (the directories won't exist anyway) so pick either option.